### PR TITLE
PHPStan fixes #2

### LINKED ISSUE #2: Advance Configuration
All variable aspects of the CCS must be configurable prior to the start of a World Finals contest. That is, the CCS may not require that any changes be made to the CCS configuration once the contest has started.

### PR DESCRIPTION
**Pull request by vmcj**
What should we do with parts where we return an item from a dependency? Explicit have an annotation so we detect changes or leave it as is as those are overly broad?

### CODE REVIEW

File: webapp/src/FosRestBundle/FlattenExceptionHandler.php [hunk 1]
```diff
@@ -44,6 +47,9 @@ public function serializeToJson(
             $type);
     }
 
+    /**
+     * @return array<string, string|int>
```
Comment (nickygerritsen) [inline webapp/src/FosRestBundle/FlattenExceptionHandler.php:51]: Another shape?

**Comment by nickygerritsen** [comment]
> What should we do with parts where we return an item from a dependency? Explicit have an annotation so we detect changes or leave it as is as those are overly broad?

I don't know what you mean.

**Comment by vmcj** [comment]
> > What should we do with parts where we return an item from a dependency? Explicit have an annotation so we detect changes or leave it as is as those are overly broad?
> 
> I don't know what you mean.

See: https://github.com/DOMjudge/domjudge/pull/2288/commits/9ecb21a832028e74c7ff84e804db761b75561a17#diff-6d7cf7ba24a3042c8310fec4a0ca3f644cd61c724beb660b4f5bfcf9e83f1291R40

Here we return the result of applying the function from one of our dependencies. In their code they don't have the annotation so I'm not sure what we want to do here?

**Comment by nickygerritsen** [comment]
> > > What should we do with parts where we return an item from a dependency? Explicit have an annotation so we detect changes or leave it as is as those are overly broad?
> > 
> > 
> > I don't know what you mean.
> 
> See: [9ecb21a#diff-6d7cf7ba24a3042c8310fec4a0ca3f644cd61c724beb660b4f5bfcf9e83f1291R40](https://github.com/DOMjudge/domjudge/commit/9ecb21a832028e74c7ff84e804db761b75561a17#diff-6d7cf7ba24a3042c8310fec4a0ca3f644cd61c724beb660b4f5bfcf9e83f1291R40)
> 
> Here we return the result of applying the function from one of our dependencies. In their code they don't have the annotation so I'm not sure what we want to do here?

Yeah not sure, we can't do much I think. Or maybe use generics

**Comment by vmcj** [comment]
So I've locally stashed the difficult parts and will look at those in the future. This should all be semi-trivial, The last commit is quite large but changing to another format also would lose information.

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**